### PR TITLE
Remove #if/def directives for UFE_PREVIEW_VERSION_NUM >= 2021 now that PR 119 is out.

### PR DESCRIPTION
**Pull request by HamedSabri-adsk**
Remove #if/def directives for UFE_PREVIEW_VERSION_NUM >= 2021

### CODE REVIEW

**Review by ppt-adsk — APPROVED** [review]
Thanks for doing this Hamed.

**Review by ppt-adsk — APPROVED** [review]

File: lib/mayaUsd/ufe/UsdSceneItem.h [hunk 1]
```diff
@@ -48,9 +48,9 @@ class MAYAUSD_CORE_PUBLIC UsdSceneItem : public Ufe::SceneItem
 
 	// Ufe::SceneItem overrides
 	std::string nodeType() const override;
-#if UFE_PREVIEW_VERSION_NUM >= 2020
+	#if UFE_PREVIEW_VERSION_NUM >= 2020
```
Comment (seando-adsk) [inline lib/mayaUsd/ufe/UsdSceneItem.h:51]: Why didn't you remove this one?

Comment (seando-adsk) [inline lib/mayaUsd/ufe/UsdSceneItem.h:51]: This should be changed into the UFE_V2 check.

**Review by seando-adsk — CHANGES_REQUESTED** [review]
Great job thanks. One minor comment.

**Comment by HamedSabri-adsk** [comment]
@ppt-adsk @seando-adsk 

Per our offline discussion,  we are unfortunately stuck with some of these #if/#defs since we still need to support UFE 1.0. :(

For now, I changed my #if/#defs from `UFE_PREVIEW_VERSION_NUM >= 2020` to `UFE_V2_FEATURES_AVAILABLE` so at least it 
is a little bit more readable :) 

I also cleaned up some bunch of unnecessary #if/defs. Will add github comments in the PR where it is appropriate tomorrow.

See 346c6f58271a37827f7b8705d10e2e6e885bdb2f

**Review by ppt-adsk — APPROVED** [review]

**Review by seando-adsk — APPROVED** [review]

**Pull request merged**